scripts/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# This adjusts the logrotate configuration to the current value of SIA_DATA_DIR.
+sed -i "s#/sia-data#$SIA_DATA_DIR#gi" /etc/logrotate.d/sia
```

Comment:
would it require some sort of restart if we change the config? dunno how that works

Reply:
No, this just sets the state of the config file's content to the appropriate one on the start of the container. If you later on log into the container and change the file again it will work just the way it would if this command wasn't here.